lexicons/tools/ozone/event/emitModerationEvent.json

```diff
@@ -1,6 +1,6 @@
 {
   "lexicon": 1,
-  "id": "com.atproto.admin.emitModerationEvent",
+  "id": "tools.ozone.emitModerationEvent",
```

Comment:
I know this introduces more work but just FYI, @bnewbold had a comment somewhere in slack recommending that we should probably follow the pattern we already have elsewhere  and sub-namespace these to `tools.ozone.event` and `tools.ozone.communication`, so on and so forth. I kinda liked that idea and got it almost working here https://github.com/bluesky-social/atproto/pull/2253/files#diff-37557b8ffd69b330154d455aee69b2fb093af41bf938cdc16300dfcf93c7a8af